octodns/record/chunked.py

```diff
@@ -18,7 +18,7 @@ def chunked_value(self, value):
             for i in range(0, len(value), self.CHUNK_SIZE)
         ]
         vs = '" "'.join(vs)
```

Comment:
@ross Note that not all providers use `" "` as a separator.
For example, constellix uses `""` without a space.

We should give the providers an opportunity to hook in and redefine the separator. Currently, providers need to "brute force" this, for example by replacing `" "` with `""`.

Reply:
Ugh, that's an annoying inconsistency. Will have to dig into/think about options for how the providers could tell the record what it needs. Since `chunked_values` is a pre-existing property there's not a clean way to pass it in there.

Reply:
The current PR is still a fix/improvement for some providers like Bind right? No harm in going forward with this and addressing a way for other providers to configure a separator?

As mentioned, those providers can apparently workaround it until then, and there is only one provider cited with the need for customizing the separator? Perhaps knowing how many other providers would need to customize the separator would be good to investigate?

Reply:
It looks like Constellix already uses `chunked_value` and I don't see anything there turning `" "` into `""`. It just seems like it's un-escaping the `;`. 

https://github.com/octodns/octodns-constellix/blob/00f51bf0ecf653d2e2bb75a0bc659cd709c8ccfe/octodns_constellix/__init__.py#L802-L807. 

That currently broken?

Reply:
Yep, it is.
```
* constellix (ConstellixProvider)
*   Update
*     <TxtRecord TXT 1800, mail._domainkey.ingo-struck.com., ['v=DKIM1\; k=rsa\; s=email\; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0uxLHfnLxBis/ZTlMC2d4oFT0rJ7SeqZN+AAW7JRtx+yZ8Fu09exnmttYpKaCSupKj3ftcvHrlu1ji4NYPlhX9qlyNwYentcFBfepp75GUTj8mtcMak72409IIkLP2fr3kHBBaDk4JR5FwMZFZsgL1IVss4WUQybKppACvEUcxeOHrftUH7Rsr2X""rCCRJopM1QV1AaQum/i3dw0ZBZgSmY9/oTtMeOco8HX/7as2M57KO8Mw+tYyx6+RyaeKOC/X9r1Da8Ayvz2P0LEojaRh4PX3LKYZiTntviO+Ed8lGrxmBXN5Jr+uqeJ+MQozoTGU+395zoAoTHpILxLr650ijQIDAQAB']> ->
*     <TxtRecord TXT 1800, mail._domainkey.ingo-struck.com., ['v=DKIM1\; k=rsa\; s=email\; p=MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA0uxLHfnLxBis/ZTlMC2d4oFT0rJ7SeqZN+AAW7JRtx+yZ8Fu09exnmttYpKaCSupKj3ftcvHrlu1ji4NYPlhX9qlyNwYentcFBfepp75GUTj8mtcMak72409IIkLP2fr3kHBBaDk4JR5FwMZFZsgL1IVss4WUQybKppACvEUcxeOHrftUH7Rsr2XrCCRJopM1QV1AaQum/i3dw0ZBZgSmY9/oTtMeOco8HX/7as2M57KO8Mw+tYyx6+RyaeKOC/X9r1Da8Ayvz2P0LEojaRh4PX3LKYZiTntviO+Ed8lGrxmBXN5Jr+uqeJ+MQozoTGU+395zoAoTHpILxLr650ijQIDAQAB']> (config)
```

However, the fix is/would be trivial. (replacing `" "` with `""`)

Reply:
> So yes, if this is seemingly the only provider, let's move forward with this one and care for the "outliers" individually as @polarathene pointed out.

Sounds good. I tend to consider bind standard compliant so hopefully it is just a quirk w/Constellix.

Reply:
I will provide a patch for constellix when this one is merged.